CMakeLists.txt

```diff
@@ -97,6 +93,9 @@ configure_file(${PROJECT_SOURCE_DIR}/src/version.h.in ${PROJECT_BINARY_DIR}/src/
 #set(GEN_PYTHON_VERSION "3" CACHE STRING "Python version")
 #set(GEN_JAVASCRIPT_BINDINGS 0 CACHE BOOL "Enable JavaScript bindings.")
 
+configure_file(${PROJECT_SOURCE_DIR}/src/cmake_config.h.in ${PROJECT_BINARY_DIR}/src/cmake_config.h @ONLY)
```

Comment:
I do not like the `cmake_config` name because it implies the special `cmake` file that aggregates information about a package for other `cmake` projects to find easily, sort-of like `pc` file but for `cmake`.

Reply:
ok, any suggestion? I'm afraid about `config.h` since there is already one in tests.

Reply:
Well, maybe rename that one to `test_config.h`.

Reply:
ok, changed